client/src/state/reducers.js

```diff
@@ -0,0 +1,21 @@
+const green = '#33DDA8'
+const red = '#FF0E0E'
+
+const toastType = (state = {}, action) => {
```

Comment:
I just added an `isActive` property to each case, and preserved the message and color state in the `REMOVE` event